lib/ref_arch_setup/bolt_helper.rb

```diff
@@ -54,7 +75,7 @@ def self.upload_file(source, destination, nodes)
       puts "Running: #{command}"
       output = `#{command}`
       success = $?.success? # rubocop:disable Style/SpecialGlobalVars
-      puts "ERROR: bolt upload failed!" unless success
+      puts "ERROR: failed to upload file #{source} to #{destination} on #{nodes.to_s}" unless success
```

Comment:
should this be `STDERR.puts` ?
i know it wasn't, prior.

Reply:
also, should we be raising, so the calling methods can handle errors?

Reply:
Discussed on the side, and filed SLV-174 to cover doing this in the future.